baseline/client/pay-info/index.js

```diff
@@ -0,0 +1,42 @@
+import "./style.css";
+import "../../../common/pay-info/style.css";
+import { getIdToken, isAdmin, Payboard } from "node_modules/pay-info";
```

Comment:
The directory that this `index.js` is in is also named `pay-info`, so if I `import "../../../common/pay-info/style.css";` I get this error:
```
WARNING in ./pay-info/index.js 28:37-45
export 'Payboard' (imported as 'Payboard') was not found in 'pay-info' (module has no exports)
```
I'm using `"node_modules/pay-info"` to disambiguate `common/pay-info` from `baseline/client/pay-info`!

Reply:
> I would have thought that it shouldn't search the local directory unless the path were relative or absolute.
I'm guessing it's because `pay-info` is in `baseline/client`, and `__dirname` is in `resolve.modules` in `webpack.common.js`.

Reply:
Renamed `baseline/client/pay-info` to `baseline/client/pay-info-page` in 9afbd026a924af33666da68bb84d7ecee1ecb389!

Reply:
> I'm guessing it's because this `pay-info` is in `baseline/client`, and `__dirname` is in `resolve.modules` in `webpack.common.js`

So it is! I just saw the list of path.join's and totally missed that lurking above them. Thanks for making the change.